torch/_inductor/scheduler.py

```diff
@@ -2619,7 +2624,10 @@ def free_buffers(self) -> None:
                 if buf.can_free():
                     V.graph.wrapper_code.codegen_free(buf.node)
             elif name in V.graph.graph_inputs:
-                storage = V.graph.graph_inputs[name].data
+                inp = V.graph.graph_inputs[name]
+                if isinstance(inp, ir.TorchBindObject):
+                    continue
```

Comment:
i think we still want to free the inputs when they are no longer needed - in your paste we never free `arg2_1`